pages/content/migration-guidance.mdx

```diff
@@ -0,0 +1,504 @@
+
+GDS will not be providing a replacement hosting service.
+
+Tenants should look to their own department, or parent department, and use existing patterns and support available for public cloud hosting services. 
+
+In deciding where to host their service next, tenants should consider the following principles from the [Central Digital & Data Office (CDDO)](https://www.gov.uk/government/organisations/central-digital-and-data-office/about#responsibilities):
+
+- Tenants should continue to use public cloud services, or internal platforms based on public cloud offerings. These continue to be the platforms of choice for government services.
+- The [Central Digital & Data Office (CDDO)](https://cddo.blog.gov.uk/) has secured Director General level agreement to help share services. Larger departments are prepared to share knowledge, patterns and platforms with the rest of government. If tenants have no existing cloud engineering capability, you could approach the CDDO, via the GOV.UK PaaS team [support form](https://admin.london.cloud.service.gov.uk/support/contact-us), to benefit from the sharing agreements.
+- When selecting a cloud platform, the tenant should assure themselves that:
+  - the public sector cloud supplier is regionally appropriate, i.e. that the data will reside in locations suitable for your workload; with GOV.UK PaaS this was predominantly the AWS eu-west-2 (London) region, but some tenancies were hosted in the AWS eu-west-1 (Ireland) region
+  - billing processes are clearly defined and understood
+  - any ‘service wrap’ provided by the department or supplier is appropriate and proportionate to the needs of your application
+  - you have suitable arrangements in place for retention or archiving of data and its safe transit through migration
+- they understand and are prepared to manage and accept any risks of using the new platform and security assurance is carried out before the new service transition date
+
+## Introduction
+
+### The purpose of this guidance
+
+To help GOV.UK PaaS tenants plan to move their services to an alternative hosting platform by 23 December 2023 and understand how the GOV.UK PaaS team can assist in the migration.
+
+### What does this mean for me and my service(s)?
+
+By 23 December 2023 all services hosted on GOV.UK PaaS will need to have migrated to an alternate hosting platform.
+
+Any services that are not hosted on another platform by that date will go offline. 
+
+All platform and tenant data will be permanently deleted by 28 February 2024. 
+
+If you miss the deadline, and you need to migrate your applications, contact us via [the normal support routes](https://admin.london.cloud.service.gov.uk/support). If you contact us before we have deleted all tenant data, we will be able to provide this data. We will not be able to provide access to the applications or their configuration on GOV.UK PaaS.
+
+### Timeline for decommissioning GOV.UK PaaS
```

Comment:
@wryobservations how about changing the title of this section to "Key Dates" I think this better describes what is is